test/images/README.md

```diff
@@ -94,8 +94,8 @@ the regular image building process. This helper image can be found in at `e2etea
 It can be used by anyone, but if you need to build your own, you can read more about it [here](windows/README.md).
 
 For Windows, in order to spawn process-isolated containers, the container OS version should closely match
-the host OS version. For this reason, we build test images for different Windows OS Versions: 1809, 1903,
-1909, 2004, 20H2. In order add support for a new Windows OS version, a new entry for that OS version will have
+the host OS version. For this reason, we build test images for different Windows OS Versions: 1809, 2004,
```

Comment:
```suggestion
the host OS version. For this reason, we build test images for different Windows OS Versions: 1809 (Windows Server 2019), 2004,
```

Should we add a note that 1809 images are used for Windows Service 2019 here?
Otherwise it might seem odd that we are removing some older Windows OS versions but not the oldest.

Reply:
Done